android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/ParagraphFieldTextView.java

```diff
@@ -15,28 +15,50 @@
 package com.liferay.mobile.screens.viewsets.defaultviews.ddl.form.fields;
 
 import android.content.Context;
+import android.text.Html;
+import android.text.InputType;
+import android.text.method.LinkMovementMethod;
 import android.util.AttributeSet;
 import com.liferay.mobile.screens.ddl.model.StringField;
 
 /**
- * @author Silvio Santos
+ * @author Victor Oliveira
```

Comment:
![:P](https://i.imgur.com/snLplqq.jpg)

😂

Reply:
😂😂😂😂 I "made this" ctrl+c ctrl+v

Reply:
😂😂😂😂😂